### DISCUSSION TITLE
openssl 3.0.4 and 1.1.1p based releases

### DISCUSSION THREAD

**Discussion by rootnk** [quictls/openssl#85] [General]
When can we expect updates?

**Comment by kaduk** [comment]
https://github.com/quictls/openssl/blob/openssl-3.0.3%2Bquic/README.md says "We will endeavor to track OpenSSL releases within a day or so"

**Reply by rootnk** [reply]
I.E just be patient.  Do you have a mailing list?

**Comment by tmshort** [comment]
These have been released.
